.github/workflows/publish-sdk.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Wormhole SDK
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.19.2'
+
+      - name: Build
+        run: |
+          cd tools && npm ci && ./build.sh && cd ..
+          tools/bin/buf generate --template buf.gen.web.yaml
+          cd ethereum && npm ci && cd ..
+          cd sdk/js && npm ci && npm run build
+
+      - name: Publish
+        uses: JS-DevTools/npm-publish@v2
```

Comment:
Let's try to publish with npm directly and don't rely on a third party tool which might not be risky sometimes.

Reply:
Done! I chose to use this because we wanted to publish based on the version defined in package.json, rather than using git tags(because we may not update the SDK code when we create a new tag). And `JS-DevTools/npm-publish@v2` can check if the version has been published. Now, we are using npm view to check if the version has been published.

Reply:
Agree that it's not convenient to publish when there are tag versions and npm package versions.